malaria-ios/ViewControllers/Info Hub and Games/Games Lists/Games/Myth vs. Fact/MythVsFactViewController.swift

```diff
@@ -70,6 +68,14 @@ class MythVsFactViewController: GameViewController {
     statementLabelInitialFrame = statementLabel.frame
   }
   
+  override func endGame() {
+    super.endGame()
+    
+    // Send a notification that the game has finished.
+    let dict = ["game" : mythVsFactGame]
```

Comment:
?

This seems like a candidate to a class "let gameFinishedEventArgs = ["game" : ...]